public/styles.css

```diff
@@ -1,12 +1,30 @@
-*{
+
+.carousel-container{
+  position: relative;
+  width: 1100px;
```

Comment:
whats up with this crazy width? does it not work otherwise?

Reply:
Haha...thats the width of the images on GameStop's website believe it or not...I thought I was a little big myself...I may have to scale them down a bit before the end.